content/flutter-publishing/publishing-to-app-store.md

```diff
@@ -69,7 +69,29 @@ Alternatively, if you wish to submit an already uploaded build for review in App
 3. To start the submission process, click **Prepare for Submission**.
 4. Check that your app metadata is up to date, and once everything is ready, click the **Submit for Review** button.
 
-Note that the application must be manually released to the App Store once Apple approves the release.
+When using the workflow editor, developers have a few different methods to choose from for publishing their app on Apple's App Store once it has been approved by Apple. Each method caters to different strategies and needs. Here’s a breakdown of the release options available and how you can configure them in App Store Connect:
```

Comment:
Would be better to explain what the default behaviour is by WFE

Reply:
Added, please review.